app/components/leagueStandingTable/leagueStandingTable.tsx

```diff
@@ -0,0 +1,26 @@
+import { Table } from "../baseComponents";
+
+import styles from "./leagueStandingTable.module.scss";
+
+export default async function LeagueStandingTable({ contestantsScores }:{contestantsScores: any}){
+    const tableColumnNames: any[] = ["Rank", "Name", "Score"];
+    const tableData = {
+        columnNames: tableColumnNames,
+        rows: contestantsScores.at(-1).contestantRoundData
+    }
+
+    tableData.rows.sort((a: any, b: any) => {
+        const aScore = a.totalScore;
+        const bScore = b.totalScore;
+        const scoreToReturn = aScore > bScore ? -1 : 1;
+        return scoreToReturn;
```

Comment:
nit: I don't think this is technically the `scoreToReturn` it is more like the `sortIndicator`
```suggestion
        const sortIndicator = aScore > bScore ? -1 : 1;
        return sortIndicator;
```